src/bci_build/package/__init__.py

```diff
@@ -834,10 +834,14 @@ def dockerfile_from_line(self) -> str:
             return ""
 
         if self.from_target_image:
-            return (
-                f"FROM {self.base_image_registry}/{self.from_target_image} AS target\n"
-                f"FROM {self._from_image} AS builder"
-            )
+            # use the full registry URL in FROM on SLE,
+            # but on TW we must not do this
```

Comment:
nit: why not?

Reply:
Because DimStar doesn't like it and keeps declining our containers: https://build.opensuse.org/request/show/1202139

Reply:
the reason is not "doesn't like it" but the reason is "we don't want to have a rebuild triggered from outside openSUSE:Factory" which seems not entirely valid because this is not a source change and Factory is on rebuild=local only. 

I keep discussing this. anyhow we should capture the actual reason in the commit message so that we can go back to it when / if this needs to be revisited.